tests/test_jwt.cpp

```diff
@@ -202,6 +202,8 @@ void test_unencrypted_pem(void **unused) {
 
   snowflake_set_attribute(sf, SF_CON_AUTHENTICATOR, SF_AUTHENTICATOR_JWT);
   snowflake_set_attribute(sf, SF_CON_PRIV_KEY_FILE, keyFilePath.c_str());
+  // explicitly unset password to esure we are using key pair auth
+  snowflake_set_attribute(sf, SF_CON_PASSWORD, "");
```

Comment:
why we need to do it?

Reply:
The key pair auth was broken but the test case passed because it was actually using user/password